draft-scenarios/clash/dragon_valley.tex

```diff
@@ -0,0 +1,171 @@
+\end{itemize}
+
+\textbf{Additionally, for 2--3 players:}
+\begin{itemize}
+  \item 1 × Center (VI-VII) Map Tile with a Dragon Utopia
+\end{itemize}
+
+\textbf{For 4 players:}
+\begin{itemize}
+  \item 2 × Center (VI-VII) Map Tiles with a Dragon Utopia
+\end{itemize}
+
+\subsection*{\MakeUppercase{Victory Conditions}}
+The game ends at the end of the Round when any of these conditions are met:
+
+\begin{itemize}
+ \item One player has defeated each other player's Main Heroes once - \textit{That player wins the game immediately.}
+ \item One player has conquered both Dragon Utopias %- \textit{That player wins the game immediately.}
+ \item At the end of Round 13 (or Round 9 for the shorter game).
+\end{itemize}
+
+\subsection*{\MakeUppercase{Victory Points}}
+If no player has achieved an immediate victory, the player with the most Victory Points (VP) wins:
+
+\begin{itemize}
+ \item 5 VP for conquering a Dragon Utopia
+ \item 4 VP for controlling 5 Mines or Settlements on Near and Center Tiles - \textit{once per game}
+ \item 4 VP for defeating an enemy's Main Hero - \textit{once per defeated Faction}
+ \item 2 VP for defeating an enemy's Secondary Hero - \textit{once per defeated Faction}
+ \item 2 VP for each enemy Main Town captured - \textit{once per captured Faction}
+ \item 1 VP for every controlled Mine or Settlement
+ \item 1 VP for every 2 Artifacts acquired during the game.
+% \item 1 VP for every Level of Experience of your Main Hero
+\end{itemize}
+
+\subsection*{\MakeUppercase{Timed Events}}
+
+\begin{itemize}
+  \item[\textbf{\nth{1}}] \textbf{Round:} All Heroes gain +1 \svgeven{movement}.
+  \item[\textbf{\nth{4}}] \textbf{Round:} Remove Black Cubes from all locations that give Resources, Morale or Cards (but not from the Grail).
+  \item[\textbf{\nth{8}}] \textbf{Round:} Repeat event of Round 4.
+  \item[\textbf{\nth{9}}] \textbf{Round:} Repeat event of Round 1.
+  \item[\textbf{End}] \textbf{of \nth{10} Round:} Player(s) with lowest XP roll 2 \svg{resource} dice.
+\end{itemize}
+
+\subsection*{\MakeUppercase{Additional Rules}}
+\begin{itemize}
+  \item Remove ``View Earth'' from the Spell Deck.
+  \item Remove ``Unexpected Reinforcements'' from the Astrologers Proclaim Deck.
+  \item Create separate Decks for Minor, Major, and Relic Artifacts.
+  \item Create separate Decks for Basic and Expert Spells.
+  \item On Starting and Far Map Tiles, players can only acquire Basic Spells and Minor Artifacts. % To-do where to acquire expert spells?
+  \item Sanctuary effect: \textbf{once per Faction} gain a \svg{morale_positive} token and draw 1 Card.
+  \item Obelisk effect: \textbf{once per Faction} roll one \svg{resource} and draw one Card.
+  
+  \MakeUppercase{Battlefield Conditions} At the star of each Round, roll 2 Attack Dice: % Perhaps after the additional rules?
+  \begin{itemize}
+    \item "-1/-1": Dense Fog -- All Ranged units gain disadvantage. %\svg{unit_ranged} "gain" or "have?" "--" or ":"
```

Comment:
```suggestion
    \item "-1/-1": Dense Fog -- All Ranged Units attack with penalty. %\svg{unit_ranged} "gain" or "have?" "--" or ":"
```
It might be obvious to TTRPG folks, but there is no game term disadvantage.